### PR TITLE
add StatisticsDisplay tab UI

### PR DESCRIPTION
**Pull request by Permas-Teo**
Statistics used is currently a placeholder. Tab currently accessed by mouse click.

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -142,6 +146,15 @@ void fillInnerParts() {
         pomodoroPlaceholder.getChildren().add(pomodoroDisplay.getRoot());
         pomodoro.setTimerLabel(pomodoroDisplay.getTimerLabel());
         pomodoro.setResultDisplay(resultDisplay);
+
+        statisticsDisplay =
```
Comment (Hardoh) [inline src/main/java/seedu/address/ui/MainWindow.java:150]: Good code formatting practice!

File: src/main/resources/view/DarkTheme.css [hunk 2]
```diff
@@ -1,10 +1,12 @@
+
 .background {
     -fx-background-color: derive(#1d1d1d, 20%);
     background-color: #383838; /* Used in the default.html file */
 }
 
+
 .label {
-    -fx-font-family: "Segoe UI Semibold";
+    //-fx-font-family: "Segoe UI Semibold";
```
Comment (artlee06) [inline src/main/resources/view/DarkTheme.css:9]: Interesting how you commented out the font

**Review by Hardoh — APPROVED** [review]
A great foundation for our statistics interface!

**Review by artlee06 — APPROVED** [review]
LGTM

**Pull request merged**